client/state/comments/test/selectors.js

```diff
@@ -76,69 +120,27 @@ describe( 'selectors', () => {
 				1
 			);
 
-			expect( res ).to.be.eql( undefined );
+			chaiExpect( res ).to.be.eql( undefined );
 		} );
 	} ); // end of getPostOldestCommentDate
 
 	describe( '#getCommentLike()', () => {
 		it( 'should provide only like statistics', () => {
 			const res = getCommentLike( state, 1, 1, 2 );
 
-			expect( res.i_like ).to.eql( false );
-			expect( res.like_count ).to.eql( 456 );
+			chaiExpect( res.i_like ).to.eql( false );
+			chaiExpect( res.like_count ).to.eql( 456 );
 		} );
 	} );
 
 	describe( '#getPostCommentsTree', () => {
 		it( 'should return the tree structure', () => {
 			const tree = getPostCommentsTree( state, 1, 1, 'all' );
-			expect( tree ).to.eql( {
-				1: {
-					children: [ 3 ],
-					data: {
-						ID: 1,
-						date: '2016-01-31T10:07:18-08:00',
-						i_like: true,
-						like_count: 5,
-						parent: false,
-					},
-				},
-				2: {
-					children: [ 4 ],
-					data: {
-						ID: 2,
-						date: '2016-01-29T10:07:18-08:00',
-						i_like: false,
-						like_count: 456,
-						parent: false,
-					},
-				},
-				3: {
-					children: [],
-					data: {
-						ID: 3,
-						date: '2017-01-31T10:07:18-08:00',
-						i_like: false,
-						like_count: 0,
-						parent: {
-							ID: 1,
-						},
-					},
-				},
-				4: {
-					children: [],
-					data: {
-						ID: 4,
-						date: '2015-01-29T10:07:18-08:00',
-						i_like: false,
-						like_count: 0,
-						parent: {
-							ID: 2,
-						},
-					},
-				},
-				children: [ 2, 1 ],
-			} );
+			expect( tree ).toMatchSnapshot();
```

Comment:
i have no idea what this means.
googling

Reply:
https://facebook.github.io/jest/docs/en/snapshot-testing.html